functions-app/config.js

```diff
@@ -57,5 +57,13 @@ module.exports = {
       Key: process.env.APPINSIGHTS_INSTRUMENTATIONKEY
     }
   },
-  Environment: getEnvironment()
+  Environment: getEnvironment(),
+  REDIS_CACHING: process.env.hasOwnProperty('REDIS_CACHING') ? toBool(process.env.REDIS_CACHING) : true,
```

Comment:
not sure you even need this?  We absolutely depend on redis for this to work....

Reply:
@GuyHarwood `REDIS_CACHING` config setting has been removed and relevant service has been updated accordingly.